### ISSUE TITLE
The CCVar constructors don't escape default string values

### ISSUE THREAD

**Issue by fnky** [baso88/SC_AngelScript#22]
Instead of `""`, the default value is just nothing.

**Comment by fnky** [comment]
Code to fix this:

```cpp
pDocumentation->RegisterObjectBehaviour( "Constructor",
    pszObjectName, asBEHAVE_FACTORY, "CCVar@ CCVar(const string& in szName, float flValue = 0, "
        "const string& in szHelpInfo = \"\", const ConCommandFlags_t flags = ConCommandFlag::None, CVarCallback@ pCallback = null)", 
    asFUNCTIONPR( CreateCVar, ( const CString&, float, const CString&, const ConCommandFlags_t, asIScriptFunction* ), CASCVar* ), asCALL_CDECL );

pDocumentation->RegisterObjectBehaviour( "Constructor",
    pszObjectName, asBEHAVE_FACTORY, "CCVar@ CCVar(const string& in szName, const string& in szValue, "
        "const string& in szHelpInfo = \"\", const ConCommandFlags_t flags = ConCommandFlag::None, CVarCallback@ pCallback = null)", 
    asFUNCTIONPR( CreateCVar, ( const CString&, const CString&, const CString&, const ConCommandFlags_t, asIScriptFunction* ), CASCVar* ), asCALL_CDECL );
```

Note the backslashes for `szHelpInfo`.

**Comment by baso88** [comment]
The fix for this issue will be included in the next update (5.13). Closing.
